libs/libRocket/Source/Core/precompiled.cpp

```diff
@@ -27,3 +27,6 @@
 
 #include "precompiled.h"
 
+// XCode's ranlib complain on empty object, adding any function prevents this
+void RocketDummyFunction() {
```

Comment:
Please make this commit in the libRocket repo. I will make this a submodule very soon.